src/components/Resource.vue

```diff
@@ -76,17 +78,16 @@
       <Token faction="bescods" transform="scale(0.15) translate(-5,26)" />
     </template>
     <image v-if="kind === 'pw'" xlink:href='../assets/resources/power-charge.svg' :height=133/346*20 width=20 transform="translate(-9.5, -13.5)" />
-
-    <!-- <SpaceShip v-else-if="kind=='ship'" class="ship" :scale="14" /> -->
-    <text x="0" y="0" v-if="count >=0 && ['o','c','k','pw','t','vp','q','ship','gf'].includes(kind) || count === '+'" :class="{plus: count === '+'}">{{count}}</text>
+    <image v-if="kind === 'pay-pw'" xlink:href='../assets/resources/power-charge.svg' :height=133/346*20 width=20 transform="translate(9.5, -13.5) scale(-1,1) " />
```

Comment:
Instead of creating a new resource `pay-pw`, you can use `pw` but with a negative number. You'd have to change the line below as well (because of the `>=0` test, maybe you can do `>=-1000` or something cleaner :p)